src/app/containers/ScheduleShiftCreateForm.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import * as moment from 'moment';
+import {
+    reduxForm,
+    ConfigProps
+} from 'redux-form';
+import { default as ScheduleShiftForm, ScheduleShiftFormProps } from '../components/ScheduleShiftForm';
+import { default as FormSubmitButton, SubmitButtonProps } from '../components/FormElements/SubmitButton';
+import { connect } from 'react-redux';
+import { RootState } from '../store';
+import { 
+    DaysOfWeek,
+    DateType 
+} from '../api/Api';
+import { 
+    ShiftCreationPayload
+} from '../api/utils';
+import { createShifts } from '../modules/shifts/actions';
+
+// wrapping generic assignment form in redux-form
+const formConfig: ConfigProps<{}, ScheduleShiftFormProps> = {
+    form: 'CreateScheduleShift',
+    onSubmit: (values, dispatch, props) => {
+        const { weekStart } = props;
+        let newShiftCreator: Partial<ShiftCreationPayload> = Object.assign({}, {...values});
```

Comment:
I'd probably call this variable something like.  `shiftCreatorConfig` / `shiftCreatorPayload` / `shiftCreatorOptions` just so that when you see it later on in the code you can tell that it's not something that is actually creating shifts